### PR TITLE
Fix virial

### PR DESCRIPTION
**Pull request by HuangJiameng**
[https://github.com/deepmodeling/dpdata/discussions/319](url)
now user can extract the virial data using "cp2k/aimd_output"

### CODE REVIEW

File: dpdata/cp2k/output.py [hunk 1]
```diff
@@ -225,6 +251,7 @@ def handle_single_log_frame(self, lines):
         info_dict['cells'] = np.asarray([self.cell]).astype('float32')
         info_dict['energies'] = np.asarray([energy]).astype('float32')
         info_dict['forces'] = np.asarray([forces_list]).astype('float32')
+        if(virial != None ): info_dict['virials'] = np.asarray([virial]).astype('float32')
```
Comment (njzjz) [inline dpdata/cp2k/output.py:254]: Per PEP-8 style, `!= None` should be written as `is not None`. Also, do not write if statement as one line.

**Review by robinzyb — COMMENTED** [review]
please add unittest for stress parser.

**Comment by HuangJiameng** [comment]
> please add unittest for stress parser.
github.com/deepmodeling/dpdata/issues/320
due to this bug, the raw files that unittest needs can't be generate yet.

**Comment by HuangJiameng** [comment]
> please add unittest for stress parser.

OK

**Review by njzjz — APPROVED** [review]

**Review by robinzyb — APPROVED** [review]

**Review by robinzyb — APPROVED** [review]

**Pull request merged**